distributed/shuffle/tests/test_rechunk.py

```diff
@@ -636,17 +636,14 @@ async def test_rechunk_with_fully_unknown_dimension(c, s, *ws, x, chunks):
         pytest.param(
             da.ones(shape=(1000, 10), chunks=(5, 10)),
             (None, 5),
-            marks=pytest.mark.skip(reason="distributed#7757"),
         ),
         pytest.param(
             da.ones(shape=(1000, 10), chunks=(5, 10)),
             {1: 5},
-            marks=pytest.mark.skip(reason="distributed#7757"),
         ),
         pytest.param(
             da.ones(shape=(1000, 10), chunks=(5, 10)),
             (None, (5, 5)),
-            marks=pytest.mark.skip(reason="distributed#7757"),
         ),
```

Comment:
It looks like these test cases still fail on CI, I'm investigating.

Reply:
Locally, these take ~7s, roughly double the time of task-based shuffling. Given the shape of the rechunk it's not surprising that task-based performs better here. I think we may want to leave these cases out or adjust their size.